portfolio/src/main/webapp/style.css

```diff
@@ -345,3 +365,26 @@ h5 {
     max-width: 100%;
   }
 }
+
+.row-r::after {
```

Comment:
What does the "r" stand for? See https://google.github.io/styleguide/jsguide.html#naming-rules-common-to-all-identifiers on variable naming